k8s/gcp-pubsub/turbinia-output-filestore.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: output
+spec:
+  capacity:
+    storage: 1T
+  accessModes:
+  - ReadWriteMany
+  nfs:
+    path: /output
+    server: <IP_ADDRESS>
```

Comment:
Just curious, do you know what the failure mode is if this gets run with kubectl as is (ie. without `<IP ADDRESS>` changed)?  Will there be an immediate error returned?

Reply:
No immediate error and it will show succeeded